public/app/controllers/hairctrl.client.js

```diff
@@ -1,45 +1,100 @@
-"use strict"
+'use strict';
 
 angular.module('hairvenApp')
-  .controller('HairCtrl', ['HairstyleService', '$scope', '$state', '$rootScope', '$localStorage',
-    function($scope, HairstyleService, $state, $rootScope, $localStorage) {
+  .controller('HairCtrl', ['$scope', 'HairstyleService', '$state',
+   '$rootScope', 'ngToast', 'Upload', 'baseUrl',
+    function($scope, HairstyleService, $state, $rootScope, ngToast, Upload, baseUrl) {
```

Comment:
Line is too long.<br>'baseUrl' is defined but never used.<br>'Upload' is defined but never used.